x-pack/plugins/secops/public/lib/ecs/index.ts

```diff
@@ -22,10 +22,13 @@ export { virtualEcsSchema } from './virtual_ecs_schema';
  *
  * A map of `EcsField.name` `->` `ECS.name`
  */
-export const mappedEcsSchemaFieldNames = {
+export const mappedEcsSchemaFieldNames: { [fieldName: string]: string } = {
```

Comment:
In other areas of the code the `Record` is usually used with `Readonly`. For example we have chunks of code like so:

```ts
const fieldTitleMapping: Readonly<Record<string, string>> = {
  'host.name': i18n.NAME,
  lastBeat: i18n.LAST_BEAT,
  'host.id': i18n.ID,
  'host.ip': i18n.IP_ADDRESS,
  'host.mac': i18n.MAC_ADDRESS,
  'host.type': i18n.TYPE,
  'host.os.platform': i18n.PLATFORM,
  'host.os.name': i18n.OS_NAME,
  'host.os.family': i18n.FAMILY,
  'host.os.version': i18n.VERSION,
  'host.architecture': i18n.ARCHITECTURE,
};
```

Just mentioning, you don't have to change it as they're the same within TypeScript just pointing out the semantic space we have.